javascript/tests/nw_tutorial/part1/duckduckgo_test_loadero.js

```diff
@@ -3,11 +3,13 @@ client => {
 
     client
         .url('https://duckduckgo.com/')
-        .waitForElementVisible('#content_homepage', 10 * 1000)
-        .setValue('#search_form_input_homepage', 'Nightwatch.js')
-        .setValue('#search_form_input_homepage', client.Keys.ENTER)
-        .assert.containsText('#r1-0 .result__title', 'Nightwatch.js')
-        .click('#r1-0 .result__title')
+        .waitForElementVisible('[data-testid=hero]', 10 * 1000)
+        .click('#searchbox_input')
+        .setValue('#searchbox_input', 'Nightwatch.js')
+        .setValue('#searchbox_input', client.Keys.ENTER)
```

Comment:
this test fails on Loadero. my experiments indicated that the issue is that when these two `.setValue` commands are called, they overwrite instead of appending one another. that is, the search bar has the text "Nightwatch.js", then it is overwritten by the "ENTER" key which erases all the text, presses ENTER and nothing happens because DuckDuckGo does not let you submit an empty field. however, if these two submissions are mentioned within the same command, then it works. for some reason this issue is not present locally, only on Loadero.
```suggestion
        .setValue('#searchbox_input', ['Nightwatch.js', client.Keys.ENTER])
```

Reply:
on the bright side this is an improvement for the script regardless